src/tools/sample/sample.cpp

```diff
@@ -7,23 +7,69 @@
 
     Provides a very simple MsQuic API sample server and client application.
 
+    The quicsample app implements a simple protocol (ALPN "sample") where the
+    client connects to the server, opens a single bidirectional stream, sends
+    some data and shuts down the stream in the send direction. On the server
+    side all connections, streams and data is accepted. After the stream is shut
+    down, the server then sends its own data and shuts down it send direction.
+    The connection only shuts down when the 1 second idle timeout triggers.
+
 --*/
 
 #include <msquic.h>
-#include <msquicp.h>
 #include <stdio.h>
 #include <stdlib.h>
 
+//
+// The (optional) registration configuration for the app. This sets a name for
+// the app (used for persistent storage and for debugging). It also configures
+// the execution profile, using the default "low latency" profile.
+//
 const QUIC_REGISTRATION_CONFIG RegConfig = { "quicsample", QUIC_EXECUTION_PROFILE_LOW_LATENCY };
+
+//
+// The application layer protocol negotiation used in the handshake.
+//
 const QUIC_BUFFER Alpn = { sizeof("sample") - 1, (uint8_t*)"sample" };
+
+//
+// The UDP port used by the server side of the protocol.
+//
 const uint16_t UdpPort = 4567;
+
+//
+// The default idle timeout period (1 second) used for the protocol.
+//
 const uint64_t IdleTimeoutMs = 1000;
+
+//
+// The amount of buffer sent over the streams in the protocol.
```

Comment:
“length”; “amount”; choose a consistent term. (“length” is better). if you choose “length”, it should also be “the buffer”.